SomerenUI/Properties/Resources.resx

```diff
@@ -146,6 +146,10 @@ Error: </value>
   <data name="No" xml:space="preserve">
     <value>No</value>
   </data>
+  <data name="NullField" xml:space="preserve">
```

Comment:
I suggest renaming the resource to "ErrorMessageNullField" for consistency.

Reply:
Sorry, also specify what the error message is about the name, please. Something like: "ErrorMessageNullNameField".